docs/dev/topology-over-raft.md

```diff
@@ -110,16 +110,17 @@ stateDiagram-v2
 A node state may have additional parameters associated with it. For instance
 'replacing' state has host id of a node been replaced as a parameter.
```

Comment:
"Tablets are never mentioned before this part of the doc, so it may be
consufing" - typo in the commit message

Reply:
fixed